JSDemos/Demos/Popup/Overview/Angular/description.md

```diff
@@ -0,0 +1,12 @@
+The **Popup** UI component is a pop-up window that overlays the current view and displays specified content.
+
+This demo illustrates the following **Popup** use cases:    
+
+- **Specify the content**       
+Define the **Popup** content within the UI component's markup.   
+
+- **Set the title**   
+Use the [showTitle](Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#showTitle) property to show or hide the **Popup** title. If the **showTitle** value is set to **true**, define the **Popup** title within the [title](/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#title) property.         
```

Comment:
This paragraphs suffers of too many repeated words. If I list all of them, this is what I get: show, title, property, show, hide, popup, title, show, title, popup, title, title, property.

 Given that property names are self-explanatory and we have a section title, we can only say this:

```suggestion
Set [showTitle](Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#showTitle) to true and use the [title](/Documentation/ApiReference/UI_Widgets/dxPopup/Configuration/#title) property to specify the caption. 
```

Reply:
yes